workspaces/src/rpc/client.rs

```diff
@@ -124,16 +121,16 @@ impl Client {
 
     pub(crate) async fn query_nolog<M>(&self, method: M) -> MethodCallResult<M::Response, M::Error>
     where
-        M: methods::RpcMethod,
+        M: methods::RpcMethod + Send + Sync,
```

Comment:
What is the motivation for adding extra constraints?

Reply:
Because `M` could be shared between threads.